### ISSUE TITLE
hello_shared module is not working with autocomplete in Android Studio/IntelliJ

### ISSUE THREAD

**Issue by wojta** [wojta/hello-kotlin#1]
Problem is with lack of provided configuration in default java plugin, it's done by hack from there http://www.sinking.in/blog/provided-scope-in-gradle/

```
configurations {
    provided
}

sourceSets {
    main {
        compileClasspath += configurations.provided
        runtimeClasspath += configurations.provided
    }
}

dependencies {
    provided "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
}
```

**Comment by wojta** [comment]
With Kotlin 1.2 this was fixed. I will post update soon.

**Comment by wojta** [comment]
Resolved by updating to new Kotlin 1.2 multiplatform support.
